helm/templates/admin/deployment.yaml

```diff
@@ -115,9 +117,11 @@ spec:
       - configMap:
           name: flyte-admin-config
         name: config-volume
+      {{- if .Values.cluster_resource_manager.enabled }}
```

Comment:
We should think about naming conventions at some point. Right now we have camel case and mixed.